google-cloud-bigquerystorage/clirr-ignored-differences.xml

```diff
@@ -3,27 +3,8 @@
 <differences>
     <!-- Allow below protobuf changes as non-breaking-->
     <difference>
-        <differenceType>8001</differenceType>
```

Comment:
please don't remove any of these existing `<difference>`s since they are needed to allow protobuf changes to be non-breaking.